src/index.ts

```diff
@@ -1,3 +1,9 @@
-export { default, ImageProps } from './components/Image';
+import { Image } from './components/Image';
+
+// Preserve default export for back-compatibility
```

Comment:
Should this be marked with `@deprecated`?

Reply:
Yeah, I'll add a named export for it too then

Reply:
Hmm, adding both default and named exports makes linter complain when importing as default:
<img width="914" alt="image" src="https://user-images.githubusercontent.com/1641218/199670451-394d35ad-6212-4adb-8d05-ac26578470b4.png">

This would basically mean a breaking change, so at this point it would make more sense to just switch to named exports completely.

However, I didn't want to introduce any breaking changes in this PR, as it is purely code-styles changes.